source/index.html.md

```diff
@@ -459,20 +461,19 @@ The last remaining task for our application is displaying the resource informati
 
 Now that we have a snazzy SMART app, it's time to test it.
 
-Next log back into the [code console](https://code.cerner.com/developer/smart-on-fhir/apps) and click on the app you've registered (My amazing SMART app). To launch your app through the code console click the "Begin Testing" button. The console will ask if the app you're launching requires a patient in context. Our app requires a patient, so select yes and choose a patient. Please note the millennium username and password, you'll need this credential when prompted. Finally, click launch and the console will redirect to your application.
+Next log back into the [code console](https://code.cerner.com/developer/smart-on-fhir/apps) and click on the app you've registered (My amazing SMART app). To launch your app through the code console click the "Begin Testing" button. If the app has a user scope, the console will ask if the app you're launching requires a patient in context. Because our app only has patient scopes, the question will be skipped and you just need to choose a patient to continue. You can choose whether to display a demographics banner for the patient. This need_patient_banner flag will be part of the launch parameters. However, our app currently does not utilize this flag when displaying the result.  Once click Next to open a "Ready to launch" popup, please note the testing username and password, you'll need this credential when prompted. Finally, click launch and the console will redirect to your application.
```

Comment:
> However, our app currently does not utilize this flag

We could re-word to use "this tutorial app" instead of "our app".

Reply:
Make sense. Will do.

Reply:
It would be awesome if you could do the following to add RED color to the text : 
```diff
- please note the testing username and password
```
This can be achieved using the github md diff api. 
The issue we have seen is that people most of the times miss this username and password. So by putting that text in red color might help in stressing the importance of it.

Reply:
Agree, with the current instruction, it is not very obvious that this is an important step for the user.